kiwi/builder/filesystem.py

```diff
@@ -128,13 +129,14 @@ def create(self) -> Result:
         )
         if self.bundle_format:
             self.result.add_bundle_format(self.bundle_format)
+        compression = self.runtime_config.get_bundle_compression(default=True)
+        if self.luks is not None:
+            compression = False
         self.result.add(
             key='filesystem_image',
             filename=self.filename,
             use_for_bundle=True,
-            compress=self.runtime_config.get_bundle_compression(
-                default=True
-            ),
```

Comment:
If we say:

```python
compress=self.runtime_config.get_bundle_compression(
    default=True if self.luks is None else False
),
```

we would still respect any setting of the runtime config if it exists

Thoughts ?

Reply:
I see the same issue at the other changed places

Reply:
ok, but I think this should be explained to the user like you did to me. Any objection that you add this explanation to the ```kiwi/kiwi.yml``` file which we ship ? In this file we explain all configuration parameters. This one would be the first were the user gets overruled under a certain condition. I don't like black magic as I suffer from obs every day and I usually go for user requests no matter how stupid they are unless they are not possible to fulfill. In this case we act as "we know better" and I think you and I made many jokes about it in the past ;) Anyway I would go with your implementation if there are comments which we can point the angry user to :)

Thanks much